src/sentry/runner/commands/presenters/optionspresenter.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+
+
+class OptionsPresenter(ABC):
+    # These messages are produced more than once and referenced in tests.
+    # This is the reason they are constants.
+    DRIFT_MSG = "[DRIFT] Option %s drifted and cannot be updated."
+    DB_VALUE = "Value of option %s on DB:"
+    CHANNEL_UPDATE_MSG = "[CHANNEL UPDATE] Option %s value unchanged. Last update channel updated."
+    UPDATE_MSG = "[UPDATE] Option %s updated. Old value: \n%s\nNew value: \n%s"
+    SET_MSG = "[SET] Option %s set to value: \n%s"
+    UNSET_MSG = "[UNSET] Option %s unset."
+    DRY_RUN_MSG = "!!! Dry-run flag on. No update will be performed."
```

Comment:
Why are the message texts in the abstract class? These should be specific to the implementation.

Reply:
I wanted to have both slack and console use these, so I put them here